src/phylum/init/cli.py

```diff
@@ -45,23 +45,61 @@
 }
 
 TOKEN_ENVVAR_NAME = "PHYLUM_TOKEN"
-PHYLUM_PATH = pathlib.Path.home() / ".phylum"
-PHYLUM_BIN_PATH = PHYLUM_PATH / "phylum"
-SETTINGS_YAML_PATH = PHYLUM_PATH / "settings.yaml"
 
-# Potential features to add:
-#  * Add logging support, a verbosity option to control it, and swap out print statements for logging
-#  * Check for valid versions by using the GitHub API to compare against actual releases
-#    * If so, also programmatically generate the SUPPORTED_TARGET_TRIPLES
-#  * Use the GitHub API ("https://api.github.com") to more programmatically get the archive URL
-#  * Add a `--list` option, to show which versions are available
-#  * Add an option to account for pre-releases
+
+def get_phylum_settings_path(version):
+    """Get the Phylum settings path based on a provided version."""
+    home_dir = pathlib.Path.home()
+    version = version_check(version)
+
+    config_home_path = os.getenv("XDG_CONFIG_HOME")
+    if not config_home_path:
+        config_home_path = home_dir / ".config"
+    phylum_config_path = pathlib.Path(config_home_path) / "phylum" / "settings.yaml"
+
+    # The Phylum config path changed following the v2.2.0 release, to adhere to the XDG Base Directory Spec
+    # Reference: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+    if Version(canonicalize_version(version)) <= Version("v2.2.0"):
+        phylum_config_path = home_dir / ".phylum" / "settings.yaml"
+
+    return phylum_config_path
+
+
+def get_phylum_bin_path(version):
+    """Get the path to the Phylum binary based on a provided version."""
+    home_dir = pathlib.Path.home()
+    version = version_check(version)
+
+    # The Phylum binary path changed following the v2.2.0 release, to adhere to the XDG Base Directory Spec
+    # Reference: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+    phylum_bin_path = home_dir / ".local" / "bin" / "phylum"
+
+    if Version(canonicalize_version(version)) <= Version("v2.2.0"):
```

Comment:
You could consider extracting this to a `use_xdg_paths(version)` (or some other name) function. And then the 2 comments about changing paths could just be one comment (or maybe be put in the docstring) on that function.

But it's not bad as is, either. Just a thought.

Reply:
changed to support suggestion.